### ISSUE TITLE
Asking for a task

### ISSUE THREAD

**Issue by poly4concept** [bancodobrasil/stop-analyzing-embed#135]
Hi all, I would love to contribute to this project, but I'm kinda new to opensource.
Following the contributing guidelines, I've forked, cloned the project, and have it running locally on my system.
I can contribute with HTML, CSS, and React. I'll appreciate it if the I'm being assigned a task

**Comment by tiagostutz** [comment]
Hi there @AkshithaReddy1899   ! 👋🏻 
Welcome to stop analyzing.!

First of all sorry sorry for the laaaaate response.

Good to know that you have already forked the repo and is aware of the Open Source best practices. 🥇 🥇 🥇 !!

Let's get something for you to tackle. Have you read the README instructions? Were you able to bring the project up and running? Need some help with that?
Please let me know as soon as you put the project running locally and then let's tackle some issues. Ok?

**Comment by AkshithaReddy1899** [comment]
Hello @tiagostutz . I have gone through every instruction in the README, and the project is running locally. I am ready for a task.

**Comment by tiagostutz** [comment]
Awesome @AkshithaReddy1899 !! ❤️ 

What about tackling https://github.com/bancodobrasil/stop-analyzing-embed/issues/72 ?
Of course, we can discuss more in detail in the issue if you wish so...

**Comment by AkshithaReddy1899** [comment]
Hi! @tiagostutz . I would love to work on it.
To my understanding,  I have to style the like mode choice card using tailwind CSS and add a close button to it as well. Is that it?

**Comment by tiagostutz** [comment]
Yeah, and there's some more things for us to discuss about, but that's a good start. 

Let's bring the discussion to the issue so all the content related to it is registered there? If you agree, please copy your last comment there and I'll answer giving some more directions. Sounds good?

**Comment by Ayush2020012016** [comment]
hello sir I will like to contribute but don't know how to start can you help me with it

**Comment by Ayush2020012016** [comment]
> Yeah, and there's some more things for us to discuss about, but that's a good start.
> 
> Let's bring the discussion to the issue so all the content related to it is registered there? If you agree, please copy your last comment there and I'll answer giving some more directions. Sounds good?

hello @tiagostutz  I will like to contribute but don't know how to start can you help me with it

**Comment by marooncoder09** [comment]
hello @tiagostutz I would like to contribute to this project  can you please help me get started

**Comment by TanayD304** [comment]
Hey @tiagostutz 4, I would like to contribute to this project. But this is my first dive in the ocean of Open source. So please can you help me initally. Thanks in advance

**Comment by tiagostutz** [comment]
Hi guys! Thanks for reaching out...

Unfortunately I'm not a maintainer of this project anymore so I cannot help with this. Maybe @andrebreves or @ralphg6 could give some help here?

Also, I have a project I've built for newcomers, if you want to give a try also: https://github.com/tiagostutz/demo-warehouse-software/issues

**Comment by Gourav2580** [comment]
is this contribution is open ore not

**Comment by sachin3825** [comment]
Hello  @tiagostutz   my name is Sachin I would like to contribute to your project I am proficient in React.js JavaScript and CSS frameworks, If there are any issues I could solve please do assign them to me. I want to start my journey in open source contribution.

**Comment by jaideep0101** [comment]
Hello @tiagostutz. I am new to open source the project . If there are some issues i could  try to solve please assign me and help to start my open source journey .

**Comment by blazete** [comment]
Hello, @tiagostutz my name is Shreyansh I would like to contribute to your project I am proficient in React.js, Nodejs, Nextjs, express, JavaScript and CSS frameworks, If there are any issues I could solve please do assign them to me. I want to start my journey in open-source contribution

**Comment by mohitbhusal11** [comment]
Hello,  @andrebreves  @ralphg6
My name is Mohit I want to contribute in open-source projects If there are any issues I could solve please do assign them to me.
I'm proficient in HTML, CSS, JavaScript, React, Bootstrap and I'm always open to learn  new technology.
Waiting for your reply sir.
